### DISCUSSION TITLE
How to Contribute

### DISCUSSION THREAD

**Discussion by ghost** [facebook/stylex#580] [Q&A]
### Describe the feature request

I would like to contribute to this project in the documentation or something related to simple examples for beginners. What do you suggest to me?

**Comment by nmn** [comment]
Propose any documentation would like to contribute and we will give you feedback.

**Reply by ghost** [reply]
@nmn Do you have any recommendations? I'm starting my studies in this technology and would like to help in some way. Something specific for styleX beginners. Thanks

**Reply by nmn** [reply]
Look through the open issues tagged with "Good first issue". Read the CONTRIBUTION.md doc. Continue to ask questions.

> I'm starting my studies in this technology

Could you clarify what you mean? Are you a student looking to make open source contributions because you've been told that helps find jobs? Or are you evaluating StyleX for your own projects?

**Reply by ghost** [reply]
@nmn I want to contribute to learn the contribution process and also to evaluate on personal projects.

**Reply by nonzzz** [reply]
@Wanderson77  Welcome to stylex. At first you need read [this](https://github.com/facebook/stylex?tab=coc-ov-file#code-of-conduct). Then if you don't know about 'stylex', I think you need to creat a relative project and using full of style's apis and read the documentation so you know what it does.
If you find any problems during use, you can provide feedback(It's also a contribute way) Or you and view the issue and find the `good first issue` or known bug and try to fix them. Finally have a fun.

**Reply by nmn** [reply]
@Wanderson77 if you're evaluating StyleX, your perspective as a new user can be helpful!

- Try building some projects with StyleX
- Read through the documentation on stylexjs.com
- Document all the things that confuse you and reports bugs you find
- Propose ways to make the documentation better

Hope that helps.
